src/components/core/PivotTable.tsx

```diff
@@ -295,29 +296,30 @@ export const getAGGridDataSource = (
         );
         return pagePromise
             .then(
-                (execution: Execution.IExecutionResponses | null) => {
-                    if (!execution) {
-                        return null;
-                    }
-                    const { columnDefs, rowData, rowTotals } = executionToAGGridAdapter(
-                        execution,
-                        resultSpecUpdated,
-                        intl,
-                        {
-                            addLoadingRenderer: 'loadingRenderer',
-                            columnDefOptions
-                        }
-                    );
-                    const { offset, count, total } = execution.executionResult.paging;
-                    // RAIL-1130: Backend returns incorrectly total: [1, N], when count: [0, N] and offset: [0, N]
-                    const lastRow = offset[0] === 0 && count[0] === 0 ? 0 : total[0];
-                    onSuccess(execution, columnDefs);
-                    successCallback(rowData, lastRow);
-                    // set totals
-                    getGridApi().setPinnedBottomRowData(rowTotals);
-
-                    return execution;
+            (execution: Execution.IExecutionResponses | null) => {
```

Comment:
Do we not indent code in (...)?

Reply:
We do, sorry auto format. I made the format more friendly to formatters